README.md

```diff
@@ -60,27 +57,11 @@ this can be accomplished are detailed in the **AWS Credentials** section below.
 
 ## Installation of MetGenC from GitHub
```

Comment:
This should be changed to "install from PyPI" text.

Reply:
Oops, done!